resources/assets/coffee/react/beatmapset-page/stats.coffee

```diff
@@ -57,7 +57,16 @@ class BeatmapsetPage.Stats extends React.Component
             el 'dt', className: 'beatmapset-stats__stat-key', Lang.get 'beatmaps.beatmapset.show.stats.source'
             el 'dd', className: 'beatmapset-stats__stat-value beatmapset-stats__stat-value--light', @props.beatmapset.source
 
-        if @props.beatmapset.tags
+        tags = _.filter @props.beatmapset.tags.split(' '), (o) -> o.length != 0
+
+        return if tags.length == 0
```

Comment:
...what. I mean removing the `tags = ` above and replacing `tags.map ...` below with

```
@props...split(' ').map (tag) =>
  return if tag.length == 0

  ...
```